usagov/usagov-2021
# USAGOV-2129: Update composer.lock to drupal-core-recommended 10.3.10 with-dependencies

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2129

## Description

Update composer.lock. This is the result of running: 
```bin/composer update --with=drupal/core-recommended:10.3.10 --with-all-dependencies```

## Type of Changes

  - Add screenshot showing what it should look like
- [x] Other: upgrade, requires rebuild

## Testing Instructions



### Change Requirements



### Validation Steps

Rebuild and poke around a bit. I think `bin/drupal-update` should cover what's needed for the rebuild. 

Drupal 10.3.9 is a security update; 10.3.10 is a bug-fix update. 10.3.7 and 10.3.8 are also bugfix releases (we were ever-so-briefly on 10.3.6). No new features are expected.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
